uno/uno.ino

```diff
@@ -36,7 +38,14 @@ void setup() {
 
 void loop() {
   if (threadSensors.shouldRun()) { threadSensors.run(); }
-  while (Serial.available()) { updateState(Serial.readString()); }
+
+  while (Serial.available()) {
+    command = Serial.readStringUntil('\n');
+  };
+
+  if (command != "") { updateState(command); };
```

Comment:
need `command = ""` after update because you will loop `updateState` on first time command received

Reply:
Ok, added